doc/sphinxman/source/glossary_psivariables.rst

```diff
@@ -120,9 +120,27 @@ PSI Variables by Alpha
    energy [H] and correlation correction components [H] for the compound
    method requested through cbs().
 
+.. psivar:: CC ROOT n DIPOLE X
```

Comment:
Nope, organizing `FOLLOW_ROOT` is an issue from the dawn of time (https://github.com/psi4/psi4private/issues/14). I think it was copied over to GH from Trac. Maybe we'll leave the equivalencies unexpressed for now until the roots get detangled. Can be taken up with CI.